nixos/attestation-server/module.nix

```diff
@@ -64,6 +94,22 @@ in
 
       serviceConfig = {
         ExecStart = "${cfg.package}/bin/AttestationServer";
+        ExecStartPre = let
+          values = concatStringsSep ", " [
+            "('emailUsername', '${cfg.emailUsername}')"
+            "('emailPassword', TRIM(readfile('%S/attestation/emailPassword'), char(10)))"
```

Comment:
In SQLite `readfile` reads a file as a BLOB which is not very useful.  However, we can use `TRIM` to convert it to a string and we have to truncate the trailing newline (`\n` = `char(10)`) anyway.

Reply:
Could you add as a comment in the source?

Reply:
Done.